docs/guide/configuration.md

```diff
@@ -103,72 +103,71 @@ The ntp-daemon binary can be configured through two channels: via command line o
-|--------------------------------------|-------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `-c <FILE>`, `--config <FILE>`       | `/etc/ntpd-rs/ntp.toml` | Which configuration file to use. When provided, the fallback locations are not used.                                                                                                                          |
-| `-l <LEVEL>`, `--log-level <LEVEL>`  | From configuration file | Override for the configuration file `log-level` parameter, see explanation there.                                                                                                                            |
-| `-p <ADDR>`, `--peer <ADDR>`         |                         | Setup a connection to the given server, overrides the peers in the configuration file. Can be given multiple times to configure multiple servers as reference.                                                |
-| `-s <ADDR>`, `--server <ADDR>`       |                         | Respond as NTP server to packets arriving to the given address, overrides server configuration in the configuration file. Can be given multiple times to attach as NTP server to multiple network interfaces. |
+| Option                              | Default                 | Description                                                                                                                                                                                                   |
+| ----------------------------------- | ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `-c <FILE>`, `--config <FILE>`      | `/etc/ntpd-rs/ntp.toml` | Which configuration file to use. When provided, the fallback locations are not used.                                                                                                                          |
+| `-l <LEVEL>`, `--log-level <LEVEL>` | From configuration file | Override for the configuration file `log-level` parameter, see explanation there.                                                                                                                             |
+| `-s <ADDR>`, `--server <ADDR>`      |                         | Respond as NTP server to packets arriving to the given address, overrides server configuration in the configuration file. Can be given multiple times to attach as NTP server to multiple network interfaces. |
```

Comment:
so the `-p` flag disappears here. That means it is no longer possible to configure sources from the CLI. Is that intentional?

Reply:
It was already removed previously.